src/qml/qgismobileapp.qml

```diff
@@ -1681,19 +1713,33 @@ ApplicationWindow {
       id: busyMessageText
       anchors.top: busyMessageIndicator.bottom
       anchors.horizontalCenter: parent.horizontalCenter
-      text: qsTr( "Loading Project" )
+      horizontalAlignment: Text.AlignHCenter
+      font: Theme.tipFont
+      color: Theme.mainColor
+      text: qsTr( "" )
```

Comment:
Does this need to be translated?

Reply:
Absolutely, they silent pauses can be interpreted in many different ways across societies... :wink: